app/src/main/java/ch/epfl/sdp/appart/utils/serializers/UserSerializer.java

```diff
@@ -24,6 +26,7 @@ private UserSerializer() {
         docData.put(UserLayout.NAME, data.getName());
         docData.put(UserLayout.PHONE, data.getPhoneNumber());
         docData.put(UserLayout.PICTURE, data.getProfileImagePathAndName());
+        docData.put(UserLayout.FAVORITE_IDS, new ArrayList<String>(data.getFavoritesIds()));
```

Comment:
Why do we recreate an Arraylist ? If getFavorites returns a List ok but otherwise I would remove it.

Reply:
In fact Emilien decided to use a Set for favorites, that's way in several places I have to convert it to a list